docker/README.md

```diff
@@ -1,6 +1,6 @@
-This folder contains Dockerfiles for Ubuntu 14.04 ROS Indigo and Ubuntu 16.04 ROS kinetic images.
+This folder contains Dockerfiles for Ubuntu 14.04 ROS Indigo and Ubuntu 16.04 ROS kinetic images. Also, oneliner script is provided to pull a specified docker image and run the respective container.
```

Comment:
I am using one-liner in documentation. So maybe we'll leave it this way for consistency?